src/Microsoft.Dnx.Runtime.Internals/RuntimeEnvironmentExtensions.cs

```diff
@@ -27,5 +30,45 @@ public static string GetShortVersion(this IRuntimeEnvironment env)
         {
             return $"{env.RuntimeType}-{env.RuntimeArchitecture}-{env.RuntimeVersion}";
         }
+
+        public static string GetRuntimeIdentifier(this IRuntimeEnvironment env)
+        {
+            if (string.Equals(env.RuntimeType, RuntimeTypes.Mono, StringComparison.Ordinal))
+            {
+                return "mono";
+            }
+
+            string os = env.OperatingSystem;
+            string ver = env.OperatingSystemVersion;
+            if (string.Equals(os, RuntimeOperatingSystems.Windows, StringComparison.Ordinal))
+            {
+                os = "win";
+
+                // Convert 6.x to the correct branding version for the RID
+                var parsedVersion = Version.Parse(ver);
+                if(parsedVersion.Major == 6 && parsedVersion.Minor == 1)
```

Comment:
I'm totally OK with adjusting it that way when you do the native->managed stuff. I didn't want to dive too far into native code right now and the manifest seemed like a quicker fix. I could remove the manifest and we just wouldn't be able to actually see `win81` and `win10` RIDs until your changes :). The manifest seems like a Good Thing generally though (however, I'm no expert there)